config/trackdirect.ini

```diff
@@ -5,9 +5,9 @@
 [database]
 
 ;; Basic database settings
-host="127.0.0.1"
+host="db"
```

Comment:
Let's write a comment here instead since "127.0.0.1" will be the thing used we setting it up a development enviroment without docker.

`;; If using docker, set host to "db"`
(or something similar)